src/common/graphql-fragments.js

```diff
@@ -3,7 +3,7 @@ import { graphql } from 'gatsby'
 export const backGroundBlur = graphql`
     fragment backGroundBlur on File {
         childImageSharp {
-            fluid(maxWidth: 1024, srcSetBreakpoints: [600, 1280]) {
+            fluid(maxWidth: 1920, srcSetBreakpoints: [600, 992]) {
```

Comment:
why reduce it to 992?

Reply:
this shouldn't be here, let me revert this 🙏